requirements.txt

```diff
@@ -12,3 +12,4 @@ PyYAML
 pathtools # supports vendored version of watchdog 0.9.0
 setproctitle
 setuptools
+pyjwt
```

Comment:
We are trying to reduce the number of base dependencies as much as possible..Could you replace this with a lazy load and a prompt to install pyjwt when neede? @kptkin do you agree?

Reply:
Hey all, thanks for checking this! I'm actually going to remove this requirement. Originally I was hitting the system-admin/api/env endpoint and using that pyjwt to decode the license from the debug bundle. But based on [comments on this pull request](https://github.com/wandb/core/pull/9826) I shouldn't be exposing that at all and will be using a redacted bundle which doesn't include the license so no need to decode.